micromasters_import/management/commands/queries/002_courserun_enrollment.sql

```diff
@@ -0,0 +1,65 @@
+--courserun migration from MicroMaster to MITxOnline
+INSERT INTO public.courses_courserun(
+    title,
+    courseware_id,
+    run_tag,
+    courseware_url_path,
+    start_date,
+    end_date,
+    enrollment_start,
+    enrollment_end,
+    expiration_date,
+    live,
+    created_on,
+    updated_on,
+    course_id)
+SELECT
+    mm_courserun.title,
+    mm_courserun.edx_course_key,
+    LEFT(CASE WHEN position('+' in mm_courserun.edx_course_key) > 0
+    THEN (regexp_match(mm_courserun.edx_course_key, '[^+]+$'))[1]
+    ELSE (regexp_match(mm_courserun.edx_course_key, '[^/]+$'))[1] END, 10) AS run_tag,
+    mm_courserun.enrollment_url,
+    mm_courserun.start_date,
+    mm_courserun.end_date,
+    mm_courserun.enrollment_start,
+    mm_courserun.enrollment_end,
+    mm_courserun.enrollment_end + INTERVAL '1 day',
+    NOT mm_courserun.is_discontinued,
+    coalesce(mm_courserun.enrollment_start, now()), --- need to confirm the default if enrollment_start is null
+    coalesce(mm_courserun.enrollment_end, now()),
+    pk_map.course_id
+FROM micromasters.courses_courserun AS mm_courserun
+JOIN public.micromasters_import_courseid AS pk_map ON mm_courserun.course_id = pk_map.micromasters_id
+LEFT JOIN public.courses_courserun AS mo_courserun ON mo_courserun.courseware_id = mm_courserun.edx_course_key
+WHERE mo_courserun.courseware_id IS NULL ---bring in course runs that don't exist in MITxOnline
```

Comment:
Do we need to add a where clause to only bring in active course runs? (mm_courserun.is_discontinued = 1). 
https://micromasters-rc.odl.mit.edu/admin/courses/courserun/?q=course%3A Peter marked these courseruns inactive, but these would still be imported if we don't filter out

Reply:
You want to only bring in the discontinued runs, because ones that aren't discontinued are the mitxonline ones, which are already over there with correct data. So you should filter to `mm_courserun.is_discontinued  = true`

Reply:
Nathan and I discussed this about an hour ago, and I think we agreed that we should only pull in courses where the `courseware_backend = 'edxorg' AND is_discontinued = false` 

The `is_discontinued` flag was used to omit mistakenly-created duplicate courses in micromasters that exist in mitxonline. 

Obviously we don't need to import courseruns that have a `courseware_backend = mitxonline`.

Reply:
@pdpinch Just to confirm if we add ```courseware_backend = 'edxorg' AND is_discontinued = false```, It would only bring in 'MITx/14.73x_1/1T2015' enrollment for your account as all other 6 would all be filtered out. 
![image](https://user-images.githubusercontent.com/3138890/182726279-e6053925-54d9-4672-876c-45aa84f10f1f.png)

I asked because you mentioned "For example, on RC my user has enrollments in 14.100x+2T2017. I expect to see that enrollment on my dashboard on mitxonline as well." in the original task.